tests/unit/test_dynamodb.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import pytest
+from moto import mock_aws
```

Comment:
Why not use only `mock_dynamodb2`?

Reply:
I remember using this in the past but it seems that this practice is no longer documented.